src/components/NewDesign/Inputs/Select/Select.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { Button, CardMedia } from '@material-ui/core'
+import { blurContent, unblurContent } from '@consts/uiUtils'
+import SelectTokenModal from '@components/NewDesign/Modals/SelectModals/SelectTokenModal/SelectTokenModal'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import icons from '@static/icons'
+import classNames from 'classnames'
+import useStyles from './style'
+
+export interface ISelectModal {
+  name?: string
+  current: string | null
+  centered?: boolean
+  tokens: Array<{ symbol: string, name: string, icon: string }> | null
+  onSelect: (chosen: number) => void
+  className?: string
+}
+export const Select: React.FC<ISelectModal> = ({
+  name = 'Select',
+  current,
+  centered,
+  tokens,
+  onSelect,
+  className
+}) => {
+  const classes = useStyles()
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null)
+  const [open, setOpen] = React.useState<boolean>(false)
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget)
+    blurContent()
+    setOpen(true)
+  }
+
+  const handleClose = () => {
+    unblurContent()
+    setOpen(false)
+  }
+
+  return (
+    <>
+      <Button
+        className={classNames(classes.button, className)}
+        color='primary'
+        variant='contained'
+        onClick={handleClick}
+        startIcon={
+          !current ? null : (
+            <CardMedia className={classes.icon} image={icons[current] ?? icons.SNY} />
```

Comment:
I think that there might be used icon from solana tokens list, that we get from props

Reply:
first of all i need to connect solana token list to Redux

Reply:
still not done